GpsMap/GpsMap_2/ViewController.swift

```diff
@@ -79,6 +88,14 @@ class ViewController: UIViewController, CLLocationManagerDelegate, UITextFieldDe
         self.initMap()
         speechRecognizer.delegate = self // マイクのデリゲード
         self.mapView.showsTraffic = true
+        // 検索バーのアクセシビリティ
+        self.serchBar!.isAccessibilityElement = true
+        self.serchBar!.accessibilityLabel = "検索フィールド"
+        self.serchBar!.accessibilityHint = "目的地の検索を行う"
+        // 現在地ボタンのアクセシビリティ
+        self.locationButton!.isAccessibilityElement = true
+        self.locationButton!.accessibilityLabel = "現在地を示すボタン"
```

Comment:
```suggestion
        self.locationButton!.accessibilityLabel = "現在地を示す"
```
ここをこうするのはどうですか？
後ろのボタンを消せば「Button」を喋っても別に違和感はないかな？と思いました。

Reply:
確かに、typeの方でもボタンと言っているのでここは変更しておきます！